modules/beopBidAdapter.js

```diff
@@ -37,6 +37,8 @@ export const spec = {
     */
   buildRequests: function(validBidRequests, bidderRequest) {
     const slots = validBidRequests.map(beOpRequestSlotsMaker);
+    const firstPartyData = bidderRequest.ortb2;
+    const psegs = (firstPartyData && firstPartyData.user) ? firstPartyData.user.psegs : undefined;
```

Comment:
are these on user.ext.psegs? hopefully permutive isn't making up mainline ortb fields

Reply:
indeed, it appears they are here 
https://github.com/prebid/Prebid.js/blob/1a5ad0cdbcc7ca772ae9fd0f2cede09f8276cd13/modules/permutiveRtdProvider.js#L195